Amethyst/Preferences/UserConfiguration.swift

```diff
@@ -289,20 +291,22 @@ final class UserConfiguration: NSObject {
             return false
         }
 
+        let useIdentifiersAsBlacklist = floatingBundleIdentifiersIsBlacklist()
+
         for floatingBundleIdentifier in floatingBundleIdentifiers {
             if floatingBundleIdentifier.contains("*") {
                 let sanitizedIdentifier = floatingBundleIdentifier.replacingOccurrences(of: "*", with: "")
                 if runningApplication.bundleIdentifier?.hasPrefix(sanitizedIdentifier) == true {
-                    return true
+                    return useIdentifiersAsBlacklist ? false : true
```

Comment:
Can you just return `!userIdentifiersAsBlacklist` instead of the ternary?

Reply:
Yeah, updating